appservice/src/main/java/nu/yona/server/messaging/rest/ActivityCommentMessageDecorator.java

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Stichting Yona Foundation This Source Code Form is subject to the terms of the Mozilla Public License, v.
+ * 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *******************************************************************************/
+package nu.yona.server.messaging.rest;
+
+import java.util.Objects;
+
+import nu.yona.server.analysis.entities.IntervalActivity;
+import nu.yona.server.analysis.service.ActivityCommentMessageDto;
+import nu.yona.server.goals.entities.Goal;
+import nu.yona.server.messaging.rest.MessageController.MessageResourceAssembler;
+import nu.yona.server.messaging.service.MessageDto;
+
+@Decorates(ActivityCommentMessageDto.class)
+public class ActivityCommentMessageDecorator implements Decorator
+{
+	@Override
+	public void decorate(MessageResourceAssembler assembler, MessageDto message)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This method has 21 lines, which is greater than the 20 lines authorized. Split it into smaller methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AS138)